src/com/puppetlabs/puppetdb/scf/storage.clj

```diff
@@ -1264,3 +1243,21 @@
   [action-for-unsupported-fn]
   (fail-on-unsupported action-for-unsupported-fn)
   (warn-on-db-deprecation))
+
+(def ^:dynamic *orphaned-path-gc-limit* 200)
+(def ^:dynamic *orphaned-value-gc-limit* 200)
```

Comment:
@kbarber do you think we should have an escape hatch here? On one hand, I can't imagine a scenario where we would even have 10 to remove, much less 200.